packages/core-sdk/resolvers/token-resolvers/resolvers/sushi/src/query/constants.ts

```diff
@@ -0,0 +1,206 @@
+export const XSUSHI_ADDRESS = "0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272";
+
+export enum ChainId {
+  MAINNET = 1,
+  ROPSTEN = 3,
+  RINKEBY = 4,
+  GOERLI = 5,
+  KOVAN = 42,
+  MATIC = 137,
+  MATIC_TESTNET = 80001,
+  FANTOM = 250,
+  FANTOM_TESTNET = 4002,
+  XDAI = 100,
+  BSC = 56,
+  BSC_TESTNET = 97,
+  ARBITRUM = 42161,
+  ARBITRUM_TESTNET = 42161,
+  MOONBEAM_TESTNET = 1287,
+  AVALANCHE = 43114,
+  AVALANCHE_TESTNET = 43113,
+  HECO = 128,
+  HECO_TESTNET = 256,
+  HARMONY = 1666600000,
+  HARMONY_TESTNET = 1666700000,
+  OKEX = 66,
+  OKEX_TESTNET = 65,
+  CELO = 42220,
+  // PALM = 11297108109, // throws because it doesn't fit in i32
```

Comment:
Let's open an issue to have chainId as BigInt in Ethereum_Network type

Reply:
see PR here: https://github.com/polywrap/monorepo/pull/747